repos/system_upgrade/el7toel8/actors/checkmultiplepackageversions/actor.py

```diff
@@ -0,0 +1,49 @@
+from leapp.actors import Actor
+from leapp.libraries.common.rpms import has_package
+from leapp.models import InstalledRPM
+from leapp.reporting import create_report, Report, Title, Summary, Severity, Flags, Remediation, RelatedResource
+from leapp.tags import IPUWorkflowTag, ChecksPhaseTag
+
+
+class CheckMultiplePackageVersions(Actor):
+    """
+    Check for problematic 32bit packages installed together with 64bit ones.
+
+    If a known problematic 32bit package is found, the upgrade will be inhibited with the detailed
+    report how to solve the problem if such a remedy exists.
+    """
+
+    name = 'multiple_package_versions'
+    consumes = (InstalledRPM,)
+    produces = (Report,)
+    tags = (IPUWorkflowTag, ChecksPhaseTag)
+
+    def process(self):
+        # package_name: remedy information
+        problem_package_map = {
```

Comment:
I'd rather see this being a global variable instead of having this here

It just blows up the function for no reason.

Reply:
Done